packages/navigation-gizmo/src/SphereScript.ts

```
@@ -1,245 +1,38 @@
+    private _xEntity: Entity;
+    private _yEntity: Entity;
+    private _zEntity: Entity;
+    private _roundEntity: Entity;
 
-  private static _vector: Vector3 = new Vector3();
 
-  private _isTriggered: boolean = false;
-  private _speedXFactor: number = 0.02;
-  private _speedYFactor: number = 0.004;
+    override onAwake() {
+        const gizmoEntity = this.entity.parent;
```

ESLint

[error] Replace `········` with `····`

(prettier/prettier)